app/modules/sightings/models.py

```diff
@@ -42,22 +42,29 @@ class SightingAssets(db.Model, HoustonModel):
     )
 
 
+class SightingTaxonomies(db.Model):
+    sighting_guid = db.Column(db.GUID, db.ForeignKey('sighting.guid'), primary_key=True)
+    taxonomy_guid = db.Column(db.GUID, primary_key=True)
+
+
 class SightingStage(str, enum.Enum):
     identification = 'identification'
     un_reviewed = 'un_reviewed'
     processed = 'processed'
     failed = 'failed'
 
 
-class Sighting(db.Model, FeatherModel):
+class Sighting(db.Model, FeatherModel, CustomFieldMixin):
     """
     Sightings database model.
     """
 
+    # Used by CustomFieldMixin class
+    CUSTOM_FIELD_NAME = 'Sighting'
```

Comment:
Unless you want the possibility of custom mappings, you can use `cls.__name__` to figure out the class' in-code name.  For example, the implementation in `app/extensions/__init__.py::CustomFieldMixin` uses this global class variable as `self.CUSTOM_FIELD_NAME`, but this can be derived from `self.__class__.__name__`.  The three locations where the `CUSTOM_FIELD_NAME` global is defined:

``` text
./app/modules/individuals/models.py:114:    CUSTOM_FIELD_NAME = 'Individual'
./app/modules/encounters/models.py:21:    CUSTOM_FIELD_NAME = 'Encounter'
./app/modules/sightings/models.py:63:    CUSTOM_FIELD_NAME = 'Sighting'
```

All use the same `CUSTOM_FIELD_NAME` value as the name of the class.  I'd argue consolidating and letting the `CustomFieldMixin` do all of the work as a single-line inheritence.

Reply:
Good suggestion. I'll do that

Reply:
yeah i like it.